agent/applier/bundlecollection/file_bundle_windows_test.go

```diff
@@ -0,0 +1,92 @@
+// +build windows
+
+package bundlecollection_test
+
+import (
+	"github.com/cloudfoundry/bosh-agent/agent/tarpath/tarpathfakes"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"os"
+
+	. "github.com/cloudfoundry/bosh-agent/agent/applier/bundlecollection"
+	"github.com/cloudfoundry/bosh-agent/agent/applier/bundlecollection/fakes"
+	fakefileutil "github.com/cloudfoundry/bosh-utils/fileutil/fakes"
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+	fakesys "github.com/cloudfoundry/bosh-utils/system/fakes"
+)
+
+//go:generate counterfeiter -o fakes/fake_clock.go ../../../vendor/code.cloudfoundry.org/clock Clock
```

Comment:
Is this something different than https://github.com/cloudfoundry/bosh-agent/tree/master/vendor/code.cloudfoundry.org/clock/fakeclock?

Reply:
Nope, it should be the same one. We copied part of the code from the file_bundle_test.go